docs/mkdocs/documentation/day0_day1_handover.md

```diff
@@ -0,0 +1,83 @@
+# Managing Day0/Day1 kmods with KMM
+
+Some kmods might be installed without KMM. In order to enhance KMM's UX we
+could, in some cases, help customers to transition the lifecycle management
+of they kmods to KMM.
+
+## Definitions
+
+### Day 0
+
+The most basic kmods that are required for a node to become “Ready” in the cluster
+
+**Examples**
+* A storage driver that is required in order to mount the rootFS as part of the boot process.
+  Vendors will usually work closely with the RHEL team to make those drivers
+  in-tree so we won’t worry about them too much here.
+* A network driver that is required for the machine to access machine-config-server
+  on the bootstrap node to pull the ignition and join the cluster
+
+### Day 1
+
+Kmods that are not required for a node to become “Ready” in the cluster but are
+preferred to be installed/updated before the node is becoming “Ready”.
```

Comment:
If the kmod is not required to be loaded before the node is `Ready`, then I think we should rather mention that the kmod is a requirement for a certain application.

Reply:
It is not always the case. For example, an outdated in-tree driver that is replaces by an up-to-date OOT network driver:
The node can become ready without the driver, but performance can be better with the new driver.
So it is not required for the infra, nor an up, but it improves the infra connection.

Reply:
Fixed.